egs2/aphasiabank/asr1/local/config.py

```diff
@@ -0,0 +1,1407 @@
+"""
```

Comment:
One of the possibility for the exact configs is to use another git to hold the information. Then we can make the code in espnet cleaner.

FYI: https://github.com/espnet/espnet/blob/master/egs2/puebla_nahuatl/asr1/local/data.sh#L48